src/quick-start.md

```diff
@@ -19,8 +19,8 @@ Currently, the chart depends on cert-manager. Make sure you have [`cert-manager`
 helm repo add kubewarden https://charts.kubewarden.io
 kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.5.3/cert-manager.yaml
 kubectl wait --for=condition=Available deployment --timeout=2m -n cert-manager --all
-helm install --create-namespace -n kubewarden kubewarden-crds kubewarden/kubewarden-crds
-helm install --wait -n kubewarden kubewarden-controller kubewarden/kubewarden-controller
+helm install --wait kubewarden-crds kubewarden/kubewarden-crds
+helm install --wait -n kubewarden --create-namespace --set "policyServer.telemetry.enabled=False" kubewarden-controller kubewarden/kubewarden-controller
```

Comment:
Shouldn't we disable metrics by default inside of the helm chart values?

Reply:
Done :)